frontend/src/app/libs/auth/store/auth/auth.reducer.ts

```diff
@@ -139,6 +140,19 @@ export function authReducer(state: AuthState = initialState, action: AuthActions
 				},
 			};
 		}
+		case AuthActionTypes.ToggleEventManagerRole: {
```

Comment:
i'm not sure we need to toggle event manager claim on current user
on admin page we can modify any user